lib/tasks/import.rake

```diff
@@ -1,10 +1,12 @@
-# frozen_string_literal: true
+ # frozen_string_literal: true
 
 require "gds_api/whitehall_export"
 
 namespace :import do
   desc "Import all documents matching an organisation, document type and optional list of document subtypes from Whitehall Publisher, e.g. import:whitehall_migration[\"cabinet-office\",\"news_article\",\"news_story,press_release\"]"
   task :whitehall_migration, %i[organisation_slug document_type document_subtypes] => :environment do |_, args|
+    include Rails.application.routes.url_helpers
```

Comment:
You may want to put this at the top following `namespace :import do` so that you don't need to include this in two separate tasks (I'm assuming that works but could be wrong)

Reply:
Done

Reply:
Reverted, as discussed in person (due to unrelated test failures)